.gitignore

```diff
@@ -26,3 +26,8 @@ venv
 
 # created from tests
 export.ris
+
+# extra benchmark data only for internal use (because of copyright)
```

Comment:
any chance we could create some synthetic data using something like faker? https://github.com/joke2k/faker